main/zoe/guide/contracts/barter-exchange.md

```diff
@@ -1,17 +1,144 @@
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/barter-exchange.js) (Last updated: 6/30/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/barterExchange.js (Last updated: 6/30/2020)
 ##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
 
-Barter Exchange takes offers for trading arbitrary goods for one another.
+Barter Exchange takes offers for trading arbitrary goods for other things
+It only accepts offers that look like:
+- `{ give: { In: amount }, want: { Out: amount}` }
 
-This Barter Exchange accepts offers to trade arbitrary goods for other
-things. It doesn't require registration of Issuers. If two offers satisfy
-each other, it exchanges the specified amounts in each side's want clause.
+You don't have to register Issuers. If two offers satisfy each other, it exchanges the  specified amounts in each side's want clause. The want amount is matched, while the give amount is a maximum. Each successful trader gets their `want` and may trade with counter-parties who specify any amount up to their specified `give`. 
```

Comment:
I realize this part is coming from the barterExchange.js comments, but you do have to register issuers, right?